flathub.json

```diff
@@ -1,3 +1,5 @@
 {
-  "only-arches": ["x86_64"]
+    "end-of-life": "Application has been renamed to io.github.aandrew_me.ytdn",
+    "end-of-life-rebase": "io.github.aandrew_me.ytdn",
+    "skip-appstream-check": true
```

Comment:
why did you add this? it's what is causing the build failure.

Reply:
Where should I add the EOL message then?

Reply:
LINE 4 is the only line I am commenting about.

Reply:
Here is the documentation

https://github.com/flathub/flathub/wiki/App-Maintenance#end-of-life

<img width="978" alt="image" src="https://user-images.githubusercontent.com/114441/224707389-7b6893a1-f8da-431a-b8ac-05e21a511136.png">

At no point it say to add     "skip-appstream-check": true

Reply:
Oh I see, ok let me remove it and add the line which you told me to add.

Reply:
> Here is the documentation
> 
> https://github.com/flathub/flathub/wiki/App-Maintenance#end-of-life
> 
> <img alt="image" width="978" src="https://user-images.githubusercontent.com/114441/224707389-7b6893a1-f8da-431a-b8ac-05e21a511136.png">
> 
> At no point it say to add "skip-appstream-check": true

Yeah I just copy pasted it, nevermind.

Reply:
I told to NOT remove the other. Because you never fixed the no aarch64 build.